### PR TITLE
dimentio: Update to latest commit

### PR DESCRIPTION
**Pull request by TheRealKeto**
This PR updates `dimentio` to its latest commit. I only tested this on iOS 12, though I'll update this message in case it doesn't work for other versions, particularly iOS 15 (though it should, considering that support has been added.)

### CODE REVIEW

File: makefiles/dimentio.mk [hunk 1]
```diff
@@ -5,7 +5,7 @@ endif
 ifneq (,$(findstring arm64,$(MEMO_TARGET)))
 
 STRAPPROJECTS      += dimentio
-DIMENTIO_COMMIT    := 7fffffff0e2a634ef3f40f4e09fcb849432fc8eb
+DIMENTIO_COMMIT    := 7ffffff7129281e3e5e0472ded3a2dea6e74b518
 DIMENTIO_VERSION   := 1:0~20210608.$(shell echo $(DIMENTIO_COMMIT) | cut -c -7)
```
Comment (CRKatri) [inline makefiles/dimentio.mk:9]: Update the date in here.

Comment (TheRealKeto) [reply makefiles/dimentio.mk]: Done!

File: makefiles/dimentio.mk [hunk 2]
```diff
@@ -5,16 +5,16 @@ endif
 ifneq (,$(findstring arm64,$(MEMO_TARGET)))
 
 STRAPPROJECTS      += dimentio
-DIMENTIO_COMMIT    := 7fffffff0e2a634ef3f40f4e09fcb849432fc8eb
-DIMENTIO_VERSION   := 1:0~20210608.$(shell echo $(DIMENTIO_COMMIT) | cut -c -7)
+DIMENTIO_COMMIT    := 7ffffff834bae1469e43ba7a5b6f1fb0766d312f
+DIMENTIO_VERSION   := 1:0~20220721.$(shell echo $(DIMENTIO_COMMIT) | cut -c -7)
```
Comment (Teutates) [inline makefiles/dimentio.mk:9]: June is 6 not 7

File: makefiles/dimentio.mk [hunk 3]
```diff
@@ -5,16 +5,16 @@ endif
...
 DEB_DIMENTIO_V     ?= $(DIMENTIO_VERSION)
 
 DIMENTIO_SOVERSION := 0
 DIMENTIO_LIBS      := -framework CoreFoundation -framework IOKit -lcompression
```
Comment (Teutates) [inline makefiles/dimentio.mk:13]: Shouldn't need libcompression anymore.

Comment (TheRealKeto) [reply makefiles/dimentio.mk:13]: Fixed in latest commit.

**Review by CRKatri — APPROVED** [review]

**Pull request merged**